idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/hetero_reactions.py

```diff
@@ -166,7 +167,9 @@ def build(self):
                           domain=Reals,
                           initialize=8e-4,
                           doc='Pre-exponential factor'
-                          '[mol^(1-N_reaction)m^(3*N_reaction -2)/s]')
+                          '[mol^(1-N_reaction)m^(3*N_reaction -2)/s]',
```

Comment:
For consistency purposes, please change all `N_reaction` occurences to `rxn_order`.